### ISSUE TITLE
Uncaught TypeError: Cannot assign to read only property '__esModule' of object '#<Object>'

### ISSUE THREAD

**Issue by patrickedqvist** [radix-ui/icons#144]
Hello Radix,

First of all thanks for this fantastic icon set.

I've set up a new Next.js 13 project along with the [new app directory](https://nextjs.org/blog/next-13#new-app-directory-beta) but whenever I try to import one Icon from the installed npm package I get the following error

```bash
error - TypeError: Cannot assign to read only property '__esModule' of object '#<Object>'
     at eval (webpack-internal:///(sc_server)/./node_modules/@radix-ui/react-icons/index.js:3:27)
     at Object.(sc_server)/./node_modules/@radix-ui/react-icons/index.js ([REDACTED])
     at __webpack_require__ ([REDACTED]/.next/server/webpack-runtime.js:33:42)
     at eval (webpack-internal:///(sc_server)/./components/ui/Icons/Icons.tsx:5:79)
     at Object.(sc_server)/./components/ui/Icons/Icons.tsx ([REDACTED]/.next/server/app/(auth)/login/page.js:708:1)
     at __webpack_require__ ([REDACTED]/.next/server/webpack-runtime.js:33:42)
     at eval (webpack-internal:///(sc_server)/./components/ui/Icons/index.tsx:5:64)
     at Object.(sc_server)/./components/ui/Icons/index.tsx ([REDACTED]/.next/server/app/(auth)/login/page.js:719:1)
     at __webpack_require__ ([REDACTED]/.next/server/webpack-runtime.js:33:42)
     at eval (webpack-internal:///(sc_server)/./app/(auth)/login/page.tsx:8:78) {
   type: 'TypeError',
   page: '/login'
}
```

Here is the small utility component that is referred to as the`components/ui/Icons/Icons.tsx`

```.tsx
import {
    ExclamationTriangleIcon,
    Cross1Icon,
    UpdateIcon,
    ChevronLeftIcon,
    ChevronRightIcon,
    TrashIcon,
    FileIcon,
    FileTextIcon,
    ImageIcon,
    GearIcon,
    DotsVerticalIcon,
    PlusIcon,
    PersonIcon,
    ArrowRightIcon,
    QuestionMarkCircledIcon,
    GitHubLogoIcon,
    TwitterLogoIcon,
    CheckIcon
} from '@radix-ui/react-icons';
import type {IconProps} from '@radix-ui/react-icons/dist/types';

import Logo from '@/components/ui/Logo'

export type Icon = IconProps

const Icons = {
    logo: Logo,
    close: Cross1Icon,
    spinner: UpdateIcon,
    chevronLeft: ChevronLeftIcon,
    chevronRight: ChevronRightIcon,
    trash: TrashIcon,
    post: FileTextIcon,
    page: FileIcon,
    media: ImageIcon,
    settings: GearIcon,
    ellipsis: DotsVerticalIcon,
    add: PlusIcon,
    warning: ExclamationTriangleIcon,
    user: PersonIcon,
    arrowRight: ArrowRightIcon,
    help: QuestionMarkCircledIcon,
    gitHub: GitHubLogoIcon,
    twitter: TwitterLogoIcon,
    check: CheckIcon,
}

export default Icons;
```

**Comment by sannajammeh** [comment]
Getting the same error. This is due to line 2 in `index.js` of the package:

```
module.exports = require('./dist/index.js');
module.exports.__esModule = true;
```

**Comment by jamesvclements** [comment]
Is there a workaround to use in the meantime?

**Comment by meluiz** [comment]
> Is there a workaround to use in the meantime?

"Well... I went into the **@radix-ui/react-icons** folder inside **node_modules** and commented out the line containing `module.exports.__esModule = true;`. It worked for me."

**Comment by t0rbik** [comment]
> Is there a workaround to use in the meantime?

Just got the same error. Overcome it by 'use client' directive in the component which uses icon.

**Comment by vladmoroz** [comment]
Hey, sorry for the delay. Just released 1.2.0 with @shuding's fix, let me know if it works.

**Comment by t0rbik** [comment]
Confirm, this does work in 1.2.0
